internal/crypto/ptls/profiles_fips_strict.go

```diff
@@ -37,11 +37,9 @@ var (
 	// insecureCipherSuiteIDs is a list of additional ciphers that should be allowed for both clients
 	// and servers when using TLS 1.2.
 	//
-	// FIPS allows the use of these specific ciphers that golang considers insecure.
-	insecureCipherSuiteIDs = []uint16{
-		tls.TLS_RSA_WITH_AES_128_GCM_SHA256,
-		tls.TLS_RSA_WITH_AES_256_GCM_SHA384,
-	}
+	// Previous versions of FIPS allowed the use of some specific ciphers that golang considers insecure.
+	// Go 1.24 does not anymore, so now this list is empty.
```

Comment:
Is there some link we could provide here to show this? Perhaps a link directly into the `stdlib` or something?

Reply:
Hmm, I'm not sure. It might be decided externally by boringcrypto.